doc/guides/writing-tests.md

```diff
@@ -231,12 +231,14 @@ assert.throws(
 
 For performance considerations, we only use a selected subset of ES.Next
 features in JavaScript code in the `lib` directory. However, when writing
-tests, it is encouraged to use ES.Next features that have already landed
-in the ECMAScript specification. For example:
+tests, for the ease of backporting, it is encouraged to use those ES.Next
+features that can be used directly without a flag in [lts/v4.x/all maintained branches](http://node.green) (See https://github.com/nodejs/lts for more information about what are those maintained branches).
```

Comment:
The lts project link should have a text. Maybe `[the LTS page](https://github.com/nodejs/lts)`.

Also nit: 80-character wrap.